### ISSUE TITLE
Error 302 When Reading Variables From Ubidots

### ISSUE THREAD

**Issue by aminhusni** [ubidots/ubidots-esp8266#38]
I can send data just fine but when it comes to reading a variable, I get this as a response:

```
Making request to Ubidots:

GET /api/v1.6/devices/ESDASG/water/lv HTTP/1.1
Host: 80
User-Agent: UbidotsESP8266/3.0.0
X-Auth-Token: A1E-xbrGx40Yi6q8lbWoL5IYrIuq2eUEx6
Content-Type: application/json

----------
Server's response:
HTTP/1.1 302 Moved Temporarily
Server: nginx
Date: Sat, 08 Feb 2020 13:36:37 GMT
Content-Type: text/html
Content-Length: 154
Location: https://80/api/v1.6/devices/ESDASG/water/lv

<html>
<head><title>302 Found</title></head>
<body bgcolor="white">
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>

----------
```

Posting is successful as below:

```
Making request to Ubidots:

POST /api/v1.6/devices/ESDASG HTTP/1.1
Host: things.ubidots.com
User-Agent: UbidotsESP8266/3.0.0
X-Auth-Token: A1E-xbrGx40Yi6q8lbWoL5IYrIuq2eUEx6
Connection: close
Content-Type: application/json
Content-Length: 96

{"Humidity":{"value":0},"Temperature":{"value":0},"Soil":{"value":100},"Sunlight":{"value":100}}
waiting for server answer ...

Ubidots' Server response:

HTTP/1.1 200 OK
Server: nginx
Date: Sat, 08 Feb 2020 13:39:22 GMT
Content-Type: application/json
Transfer-Encoding: chunked
Connection: close
Vary: Accept-Encoding
Vary: Cookie
Allow: GET, POST, HEAD, OPTIONS
x-frame-options: SAMEORIGIN
X-XSS-Protection: 1;mode=block

8f
{"soil": [{"status_code": 201}], "temperature": [{"status_code": 201}], "sunlight": [{"status_code": 201}], "humidity": [{"status_code": 201}]}
0
```

**Comment by jotathebest** [comment]
To be solved at #39 , pending to deploy.

**Comment by jotathebest** [comment]
solved at #39
